Makefile

```diff
@@ -58,7 +70,10 @@ install-rancher: ## Install Rancher via Helm on the k8s cluster
 		--set replicas=1 \
 		--set rancherImageTag=v${RANCHER_VERSION} \
 		--wait
-	kubectl rollout status deployment rancher -n cattle-system --timeout=300s
+	@$(KUBECTL_ROLLOUT_DEPLOYMENT) && \
+	kubectl_rollout_deployment_with_timeout 300s rancher cattle-system && \
+	kubectl_rollout_deployment_with_timeout 300s fleet-controller cattle-fleet-system && \
+	kubectl_rollout_deployment_with_timeout 300s rancher-webhook cattle-system
```

Comment:
Perhaps you could create another macro function that checks all the 3 deployment statuses and simply call the new macro in all targets?